source/cluster-man/dovecot-cluster.7.rst.in

```diff
@@ -0,0 +1,50 @@
+.. _man-7_dovecot-cluster:
+
+===============
+dovecot-cluster
+===============
+
+
+NAME
+====
+
+dovecot-cluster - How dovecot cluster works
+
+
+DESCRIPTION
+===========
+
+Cluster's main job is to direct users into specific backends to distribute the load and provide high availability for the cluster.
+Same user's concurrent connections should go to the same backend. Otherwise the IMAP connections' states may look different, which may confuse user/client.
+For high availability, there needs to be an eventual automatic failover to move users from broken backends to working ones.
+
+Cluster tries to optimize where a new connection goes. For storage optimization, user should be redirected to the backend where it already has its indexes in metacache.
+For cluster optimization, users should be distributed in a way that all backends have approximately equal load (IO, memory, CPU).
+
+For best high availability users should be moved away from a broken backend immediately. The backend may be broken only for some seconds,
```

Comment:
- For [x], there needs [...];
- for [y], user should [...]; <- also maybe "users"
- for [z], users should [...]; but then
- for [a] `(no comma)` users should [...];

Disregard the stylistic repetition but I think either consistently no comma or consistently commas everywhere.